docs/algosdk/future/index.rst

```diff
@@ -3,6 +3,5 @@ future
 .. toctree::
    :maxdepth: 10
 
-   template
```

Comment:
I think there is also a reference in `algosdk/index.rst`: https://github.com/algorand/py-algorand-sdk/blob/develop/docs/algosdk/index.rst

Reply:
👍